addons/repair/CfgVehicles.hpp

```diff
@@ -334,4 +334,9 @@ class CfgVehicles {
         GVAR(canRepair) = 1;
         transportRepair = 0;
     };
+
+    class Quadbike_01_base_F;
+    class B_Quadbike_01_F: Quadbike_01_base_F {
+        GVAR(hitpointPositions[]) = {"HitEngine", "0, 0.5, -0.7", "HitFuel", "0, 0, -0.5"};
```

Comment:
I dislike this format. I'd prefer a 2d array for this instead. 

`{ {"HitEngine", {0, 0.5, -0.7} }, {"HitFuel", {0, 0, -0.5} } };`